### PR TITLE
metastore: add au.hash

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/metastore.json [hunk 1]
```diff
@@ -26,6 +26,10 @@
             "32bit": {
                 "url": "https://github.com/rasa/metastore/releases/download/v$version/metastore.exe"
             }
+        },
+        "hash": {
+            "url": "https://github.com/rasa/metastore/releases/latest",
```
Comment (Ash258) [inline bucket/metastore.json]: Use specific tag release for supporting downgrading with `scoop install metascore@<version>`

File: bucket/metastore.json [hunk 2]
```diff
@@ -18,6 +13,11 @@
             "hash": "a3ecd6ce64750cb3925bb5cce1785ffdc1ee7ae7f5d50c879396e246c68b0a2d"
         }
     },
+    "bin": "metastore.exe",
+    "checkver": {
+        "url": "https://github.com/rasa/metastore/releases/latest",
```
Comment (Ash258) [inline bucket/metastore.json]:
```suggestion
        "github": "https://github.com/rasa/metastore/",
```

File: bucket/metastore.json [hunk 3]
```diff
@@ -26,6 +26,10 @@
             "32bit": {
                 "url": "https://github.com/rasa/metastore/releases/download/v$version/metastore.exe"
             }
+        },
+        "hash": {
+            "url": "https://github.com/rasa/metastore/releases/tag/v$version",
+            "regex": "$sah256 \\*bin/$basename"
```
Comment (Ash258) [inline bucket/metastore.json]:
```suggestion
            "regex": "$sha256 \\*bin/$basename"
```

**Comment by rasa** [comment]
Thank you @linsui !

**Pull request merged**